pyrobosim/test/utils/test_pose_utils.py

```diff
@@ -276,6 +323,28 @@ def test_is_approx():
     assert exc_info.value.args[0] == "Expected a Pose object."
 
 
+def test_is_approx_degrees():
+    """Test approximate equivalence functionality for pose in degrees"""
+    pose1 = Pose(
+        x=1.0, y=2.0, z=3.0, roll=np.pi / 2, pitch=0.0, yaw=-90.0, angle_units="degrees"
+    )
+    pose2 = Pose(
+        x=1.0 + 1e-4,
+        y=2.0,
+        z=3.0,
+        roll=np.pi / 2,
```

Comment:
Wait this is combining units, so it's wrong. Is that why the tolerances had to go up?

Reply:
Fix and thank you for pointing this out.